src/presentational/Params.tsx

```diff
@@ -0,0 +1,74 @@
+import {Codec, TypeDef} from '@polkadot/types/types';
+import {Account} from 'presentational/Account';
+import {StyleSheet, View} from 'react-native';
+import {Text} from '@ui-kitten/components';
+import Identicon from '@polkadot/reactnative-identicon';
+import Padder from 'presentational/Padder';
+import {isU8a, u8aToString} from '@polkadot/util';
+import {Balance} from '@polkadot/types/interfaces';
+import React from 'react';
+import {standardPadding} from 'src/styles';
+import {useFormatBalance} from '../hook/useFormatBalance';
+
+export interface Param {
+  name: string;
+  type: TypeDef;
+  value?: Codec;
+}
+
+export function Params({data}: {data: Param[]}) {
```

Comment:
The name and purpose of this component it's a bit confusing, `Params` is a generic name, but the component displays data related to an account, balance, etc. 

Is this meant to be reused?

Reply:
yes, it's a bare implementation of this package. https://github.com/polkadot-js/apps/tree/master/packages/react-params/src/Param

the purpose is to read the params of a Call object and show info about it.

maybe we can have a `packages` folder and have this moved there for now before we complete it?

Reply:
I see.. I guess we could have our own packages (if we cannot use them directly from the polkadot-js libraries), that's a good idea. Nevertheless, I think we should use names that reflect better what we are building (Features/UI).. As someone new to the code, If I want to build something where I need to display account and balance info for example, I would never guess that there is an UI element called `Params` that I can use for that xD. They use really bad names because they are based in data structures and no UI/Features

Reply:
unfortunately, it seems they wrote their packages with only the web in mind. there is not a good separation between their views and logic which makes their code unusable for the most part for us. 😞

Reply:
1b5e4cf moved to packages folder here. it's not a real package just inside a folder named packages and I called it `CallInspector` couldn't come up with a better name 😅 . maybe when we brushed up this and make sure it support all forms of params then extract it as a standalone package.